requirements.txt

```diff
@@ -15,4 +15,5 @@ qimage2ndarray>=1.10
 aiortc>=1.5
 qasync>=0.26
 onnxruntime-gpu>=1.16.1
-onnxruntime-openvino
\ No newline at end of file
+onnxruntime-openvino
+pyjoystick
```

Comment:
We need to make sure to include this dependency with the contributors and license file.